digitalpublicgoods/notesmaster.json

```diff
@@ -0,0 +1,119 @@
+{
+  "name": "NotesMaster",
+  "clearOwnership": {
+    "isOwnershipExplicit": "Yes",
+    "copyrightURL": "https://notesmaster.com/termsofuse"
+  },
+  "platformIndependence": {
+    "mandatoryDepsCreateMoreRestrictions": "No",
+    "isSoftwarePltIndependent": "",
+    "pltIndependenceDesc": ""
+  },
+  "documentation": {
+    "isDocumentationAvailable": "Yes",
+    "documentationURL": [
+      "System requirements - https://notesmaster.com/systemrequirements"
+    ]
+  },
+  "NonPII": {
+    "collectsNonPII": "No",
+    "checkNonPIIAccessMechanism": "",
+    "nonPIIAccessMechanism": ""
```

Comment:
Content and Data that do not contain Personal Identifiable Information (PII) should fall in this category. Given that NotesMaster is seeking DPG status as educational content, it should provide a `nonPIIAccessMechanism`

Reply:
Product owner clarified data/content can be exported in the open formats: JSON, XML, CSV.

Reply:
Resolved in commit c79185a